release-notes/1.0/known-issues-1.0.0.md

```diff
@@ -0,0 +1,232 @@
+
+The workaround to address the above problem is one of the following:
+
+Load the assembly using `LoadFromAssemblyName` instead of Assembly.Load, OR
+Create a custom `AssemblyLoadContext` and override its Load method to resolve the assembly.
+
+<https://github.com/dotnet/coreclr/issues/5837#issuecomment-226657996>
+
+## Nano Server TP5
+
+When working on TP5 of Nano server, users will encounter the following error if they try to run either portable or self-contained application:
+
+```
+Failed to load the dll from [C:\hwapp_s\bin\Debug\netcoreapp1.0\win10-x64\hostpolicy.dll], HRESULT: 0x8007007E
+An error occurred while loading required library hostpolicy.dll from [C:\hwapp_s\bin\Debug\netcoreapp1.0\win10-x64]
+```
+
+If you’re using Nano Server Technical Preview 5 with .NET Core CLI, due to a bug in the product, you will need to copy binaries from `c:\windows\system32\forwarders`. The destination depends on the type of deployment that you are choosing for your application.
+
+For portable applications, the forwarders need to be copied to the shared runtime location. The shared runtime can be found wherever you installed .NET Core 1.0.0 (by default, it is `C:\Program Files\dotnet`) under the following path: `shared\Microsoft.NETCore.App\<version>\`.
+
+For self-contained applications, the forwarders need to be copied over into the application folder, that is, whereever you put the published output.
+
+This process will ensure that that the dotnet host finds the appropriate APIs it needs. If your Nano Server Technical Preview 5 build is updated or serviced, please make sure to repeat this process, in case any of the DLLs have been updated as well.
+
+## dotnet does not support UTF-16 BOM
+
+If global.json file has an UTF-16 BOM, dotnet commands will fail due to not supporting UTF-16 BOM with the error:
+
+```
+A JSON parsing exception occurred: * Line 1, Column 2 Syntax error: Malformed token
+```
+
+If you save global.json with an UTF-16 BOM (Byte Order Mark), dotnet commands will error out with the following error:
+
+A JSON parsing exception occurred: * Line 1, Column 2 Syntax error: Malformed token
+
+This situation may arise if using tools that by default produce UTF-16 files with the BOM, such as PowerShell's Out-File cmdlet.
+
+**The workaround** is to either remove the BOM or change it to UTF-8 BOM, as these are the currently supported. Visual Studio by default uses UTF-8. If you are using PowerShell, you can specify the encoding for the Out-File cmdlet with the -Encoding argument.
+
+<https://github.com/dotnet/core-setup/issues/185>
+<https://github.com/dotnet/cli/issues/2159>
+
+## dotnet restore can fail on Fedora 23 with NSS 3.24 installed with timeouts and SSL connection errors
+
+At this time, the workaround is to downgrade NSS.
+
+```
+sudo dnf downgrade nspr nss-util nss-softokn-freebl nss-softokn nss-sysinit nss nss-tools
+```
+
+<https://github.com/dotnet/cli/issues/3676>
+
+
+
```

Comment:
We appear to be missing both of these:
https://github.com/dotnet/corefx/issues/9235
https://github.com/dotnet/corefx/issues/9240
Not sure if that's by choice or an accident.